crates/bws/src/main.rs

```diff
@@ -151,141 +114,52 @@ async fn process_commands() -> Result<()> {
-                .get(&ProjectGetRequest { id: project_id })
-                .await?;
-            serialize_response(project, cli.output, color);
-        }
+        } => command::project::get(client, project_id, cli.output, color).await,
 
         Commands::Project {
             cmd: ProjectCommand::Create { name },
         }
         | Commands::Create {
             cmd: CreateCommand::Project { name },
-        } => {
-            let project = client
-                .projects()
-                .create(&ProjectCreateRequest {
-                    organization_id,
-                    name,
-                })
-                .await?;
-            serialize_response(project, cli.output, color);
-        }
+        } => command::project::create(client, organization_id, name, cli.output, color).await,
 
         Commands::Project {
             cmd: ProjectCommand::Edit { project_id, name },
         }
         | Commands::Edit {
             cmd: EditCommand::Project { project_id, name },
         } => {
-            let project = client
-                .projects()
-                .update(&ProjectPutRequest {
-                    id: project_id,
-                    organization_id,
-                    name,
-                })
-                .await?;
-            serialize_response(project, cli.output, color);
+            command::project::edit(client, organization_id, project_id, name, cli.output, color)
+                .await
         }
 
         Commands::Project {
             cmd: ProjectCommand::Delete { project_ids },
         }
         | Commands::Delete {
             cmd: DeleteCommand::Project { project_ids },
-        } => {
-            let count = project_ids.len();
-
-            let result = client
-                .projects()
-                .delete(ProjectsDeleteRequest { ids: project_ids })
-                .await?;
-
-            let projects_failed: Vec<(Uuid, String)> = result
-                .data
-                .into_iter()
-                .filter_map(|r| r.error.map(|e| (r.id, e)))
-                .collect();
-            let deleted_projects = count - projects_failed.len();
-
-            if deleted_projects > 1 {
-                println!("{} projects deleted successfully.", deleted_projects);
-            } else if deleted_projects == 1 {
-                println!("{} project deleted successfully.", deleted_projects);
-            }
-
-            if projects_failed.len() > 1 {
-                eprintln!("{} projects had errors:", projects_failed.len());
-            } else if projects_failed.len() == 1 {
-                eprintln!("{} project had an error:", projects_failed.len());
-            }
-
-            for project in &projects_failed {
-                eprintln!("{}: {}", project.0, project.1);
-            }
-
-            if !projects_failed.is_empty() {
-                process::exit(1);
-            }
-        }
+        } => command::project::delete(client, project_ids).await,
```

Comment:
I think it's time to get rid of the deprecated methods (Preferably in a separate PR).

This would allow us to do something like

```rust
        Commands::Project { cmd } => match cmd {
            ProjectCommand::List => {
                command::project::list(client, organization_id, cli.output, color).await
            }
            ProjectCommand::Get { project_id } => {
                command::project::get(client, project_id, cli.output, color).await
            }
            ProjectCommand::Create { name } => {
                command::project::create(client, organization_id, name, cli.output, color).await
            }
            ProjectCommand::Edit { project_id, name } => {
                command::project::edit(client, organization_id, project_id, name, cli.output, color)
                    .await
            }
            ProjectCommand::Delete { project_ids } => {
                command::project::delete(client, project_ids).await
            }
        },
```

And you could lift in the inner match to `command/project.rs`. Which should nicely align with the other refactors you've done here.

Reply:
We have a ticket for the deprecated commands already ([SM-1175](https://bitwarden.atlassian.net/browse/SM-1175)), but lifting the inner match is a great idea. I'll add that to SM-1175.

Reply:
PR: https://github.com/bitwarden/sdk/pull/836